app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({
+            name: self
+        })
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        new_person = Person(name=person["name"], age=person["age"])
+        person_list.append(new_person)
+
+    for person in people:
+        person_instance = Person.people[person["name"]]
```

Comment:
Accessing `Person.people[person['name']]` assumes that the person exists in the `people` dictionary. If a name is not found, this will raise a `KeyError`. Consider adding error handling to manage cases where the name might not exist.